### DISCUSSION TITLE
[standalone] ngrx feature store generator missing path

### DISCUSSION THREAD

**Discussion by alexanderPastornicky** [nrwl/nx#22611] [Feature Requests]
### Current Behavior

Hi, we have following a project structure called 'portals' in NX docs, so we have /libs/shell as the main shell where are routes to each application shell.

NOTE: project structure/names/ are simplified for illustration 
```
libs/
├── application1/
│   ├── data-access-aaa
│   └── shell/
│       └── routes.ts
├── application2/
│   └── shell/
│       └── routes.ts
└── shell/
    └── routes.ts
```

The problem is that I am not able to generate feature-store in data-access library in application1, so if I provide `parent` to ngrx feature generator libs/application1/shell/routes.ts, it will generate +state in shell library

### Expected Behavior

I should be able to choose path where +state should be generated. For example in /libs/shared/data-access library. 

### Steps to Reproduce

1.

### Nx Report

```shell
>  NX   Report complete - copy this into the issue template

   Node   : 18.18.0
   OS     : win32-x64
   npm    : 10.2.0

   nx                 : 17.2.0-beta.15
   @nx/js             : 17.2.0-beta.15
   @nx/jest           : 17.2.0-beta.15
   @nx/linter         : 17.2.0-beta.15
   @nx/eslint         : 17.2.0-beta.15
   @nx/workspace      : 17.2.0-beta.15
   @nx/angular        : 17.2.0-beta.15
   @nx/cypress        : 17.2.0-beta.15
   @nx/devkit         : 17.2.0-beta.15
   @nx/eslint-plugin  : 17.2.0-beta.15
   @nrwl/tao          : 17.2.0-beta.15
   @nx/web            : 17.2.0-beta.15
   @nx/webpack        : 17.2.0-beta.15
   typescript         : 5.2.2
   ---------------------------------------
   Community plugins:
   @ngrx/component-store : 17.0.1
   @ngrx/effects         : 17.0.1
   @ngrx/entity          : 17.0.1
   @ngrx/router-store    : 17.0.1
   @ngrx/schematics      : 17.0.1
   @ngrx/store           : 17.0.1
   @ngrx/store-devtools  : 17.0.1
```

### Operating System

- [ ] macOS
- [X] Linux
- [X] Windows
- [ ] Other (Please specify)

**Comment by distante** [comment]
Not sure why this was moved to a discussion and completely ignored :) 

The `shared/data-access` pattern is, IMO, a widely used one.
